lib/watir/rails.rb

```diff
@@ -138,11 +138,25 @@ def find_available_port
       end
 
       def run_default_server(app, port)
+        loaded = false
```

Comment:
Instead of using a boolean here we could just `return` from each `begin`...`rescue` block like this:
```ruby
begin
  require "rack/handler/thin"
  return Rack::Handler::Puma.run(...)
rescue LoadError
end
```

If loading fails then `LoadError` will be catched, but otherwise method will exit.